### DISCUSSION TITLE
Can I hide myself from Contributors list on a repo main page?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#53502] [New to GitHub]
(without deleting account, without asking/blocking repo owner)

**Comment by airtower-luna** [comment]
If the repository is owned by a user, blocking them should work. The [documentation on blocking users](https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account) lists the effects a block has, and two of the points are:

> * Your contributions to the blocked user's repositories are no longer counted as contributions for you
> * You are removed as a collaborator on their repositories

I'm not sure if there's anything you can do if the repository is owned by an organization.

**Reply by ghost** [reply]
Thank you very much for finding a solution to my question, I update it because I don't want to hide myself by blocking the user

I guess what I want do not exist and for it to exist I should make a request to github
